src/components/Layout/Header.tsx

```diff
@@ -126,12 +126,15 @@ class Header extends React.Component<HeaderProps, HeaderState> {
 
   public handleLogout = () => {
     this.setState({ anchorEl: null })
-    this.cookies.remove('DevCenter.token', {
-      path: '/',
-      domain: window.location.hostname,
-    })
-    this.cookies.set('DevCenter.firstName', { path: '/' })
-    this.cookies.set('DevCenter.email', { path: '/' })
+
+    ;['DevCenter.token', 'DevCenter.firstName', 'DevCenter.email'].forEach(
```

Comment:
Can you remove the semi colon? Or put it after the this.setState()

Reply:
The auto-formatter actually put it there. I think its semantically required so it gets interpreted correctly. 

It would be nice to put it on the previous line so it doesnt look as ugly but I don't know how to get the auto-formatter to do that